cypress/component/DarCollectionReview/dar_collection_review.spec.jsx

```diff
@@ -1,6 +1,6 @@
-/* eslint-disable no-undef */
+ 
```

Comment:
minor nit: remove spacing / empty lines before the first import?

Reply:
Grrr.  My IDE shouldn't have removed the eslint override.  It's still needed.  I'll be adding it back in to the files it was removed from.